test/parallel/test-crypto-classes.js

```diff
@@ -18,7 +18,7 @@ const TEST_CASES = {
   'Verify': ['RSA-SHA1'],
   'DiffieHellman': [1024],
   'DiffieHellmanGroup': ['modp5'],
-  'ECDH': ['prime256v1'],
+  //'ECDH': ['prime256v1'],
```

Comment:
was this intentional?

Reply:
Yeah, there's no longer a `createECDH` function so this test fails if I uncomment this.

If I were to develop this proposal further (which I'm more than happy to do if there's interest!), I'd refactor all the other crypto functions in the same way and update this test accordingly.